src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Collection/DictionaryOfStringTValueConverter.cs

```diff
@@ -62,7 +62,7 @@ protected override void CreateCollection(ref ReadStack state)
                 {
                     string key = GetKeyName(enumerator.Current.Key, ref state, options);
                     writer.WritePropertyName(key);
-                    converter.Write(writer, enumerator.Current.Value, options);
+                    converter.Write(writer, enumerator.Current.Value!, options);
```

Comment:
nit: Last one - add TODO with link to issue.

Reply:
Fixed.